web_src/js/features/comp/LabelEdit.js

```diff
@@ -3,21 +3,24 @@ import {initCompColorPicker} from './ColorPicker.js';
 
 export function initCompLabelEdit(selector) {
   if (!$(selector).length) return;
+  initCompColorPicker();
+
   // Create label
-  const $newLabelPanel = $('.new-label.segment');
   $('.new-label.button').on('click', () => {
-    $newLabelPanel.show();
-  });
-  $('.new-label.segment .cancel').on('click', () => {
-    $newLabelPanel.hide();
+    $('.new-label.modal').modal({
+      onApprove() {
+        $('.new-label.form').trigger('submit');
+      }
+    }).modal('show');
+    return false;
   });
 
-  initCompColorPicker();
-
+  // Edit label
   $('.edit-label-button').on('click', function () {
     $('.edit-label .color-picker').minicolors('value', $(this).data('color'));
     $('#label-modal-id').val($(this).data('id'));
     $('.edit-label .new-label-input').val($(this).data('title'));
+    $('.edit-label .new-label-exclusive').prop('checked', $(this).data('exclusive'));
```

Comment:
Since the `checked` is a bool property, its value  should also be  a valid  bool value.

I guess string values like `prop('checked', 'false')` or `prop('checked', '0')` will   cause bug?

Reply:
I wasn't really sure what you had in mind here, I changed it to `.hasAttribute`  given the previous comment to discourage `.data` usage.

Reply:
I am not sure about what `data-exclusive="{{.Exclusive}}"` would output.

If it outputs: `data-exclusive="true"` and `data-exclusive=""` then it's OK

But if it outputs: `data-exclusive="1"` and `data-exclusive="0"` then I guess there would be a bug? 

I haven't confirmed it by myself, just a guess. Feel free to ignore this comment if there is no problem.

Reply:
More details, for jQuery: IIRC, `$().prop('checked', val)` behavior is:
* val = 'non-empty', val=non-zero-number,  value=true: then checked=true
* val = '', val=0,  value=false: then checked=false
* but the `data("exclusive")` always returns a string

So, if `data-exclusive` attribute (string) is not empty when `.Exclusive` is falsey, the `prop` will make `checked=true` for the checkbox. Just a guess.

Reply:
It was working correctly. But now it's using `.hasAttribute` so we don't have to think about what jQuery is doing at all.